insights/client/constants.py

```diff
@@ -13,15 +13,18 @@ class InsightsConstants(object):
     default_conf_file = '/etc/insights-client/insights-client.conf'
     user_agent = os.path.join(app_name, package_info["VERSION"])
     log_dir = os.path.join(os.sep, 'var', 'log', app_name)
+    simple_find_replace_dir = '/etc/redhat-access-insights'
```

Comment:
Can you explain why you chose `simple_find_replace` for the name?

Reply:
Once upon a time it was said that renaming the RPM would be a simple find and replace 😏